knowledge-base/actions/arhia/action-check-typescript/action-security.yml

```diff
@@ -0,0 +1,12 @@
+name: 'Action Check Typescript errors'
+github-token:
+  action-input:
+    input: repo-token
+    is-default: false
+  permissions:
```

Comment:
@nasablackberry be more specific while specifying permissions, for instance
- pull-requests-reason: to create PR review of analyzed code 
- issues-reason: to comment errors found from code analysis

There is also `checks: write` permission, view [here](https://github.com/Arhia/action-check-typescript/blob/d1a7bbe877766c64f1703a48cd707b13488fdae3/src/main.ts#L239), and there is condition attached to it `args.useCheck` -> `use-check` view [here](use-check).
so you have to add the condition like this for this permission:
`checks-if: ${{ contains(with, 'use-check') && with['use-check'] == 'true'}}`